src/sentry/features/__init__.py

```diff
@@ -200,6 +200,7 @@
 default_manager.add("projects:similarity-indexing-v2", ProjectFeature)
 default_manager.add("projects:similarity-view", ProjectFeature)
 default_manager.add("projects:similarity-view-v2", ProjectFeature)
+default_manager.add("projects:sourcemapcache-processor", ProjectFeature, True)
```

Comment:
what does the `True` mean here?

Reply:
Use Flagr to control the value of feature flag. See: https://develop.sentry.dev/feature-flags/